Integrations/integration-FireEyeHX.yml

```diff
@@ -3043,7 +3134,112 @@ script:
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.display_name
+      description: The indicator display name
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.name
+      description: The generated name
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.created_by
+      description: Who created the indicator
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.url
+      description: The data URL
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.create_text
+      description: The indicator create text
+    - contextPath: FireEyeHX.Indicators.data.platforms
+      description: List of families of operating systems.
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.create_actor._id
+      description: The ID of the actor
+      type: number
+    - contextPath: FireEyeHX.Indicators.data.create_actor.username
+      description: Actor user name
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.signature
+      description: 'Signature of indicator '
+      type: string
+    - contextPath: FireEyeHX.Indicators.data._revision
+      description: Indicator revision
+      type: string
+    - contextPath: FireEyeHX.Indicators.data._id
+      description: FireEye unique indicator ID.
+      type: string
+    - contextPath: FireEyeHX.Indicator.data.description
+      description: Indicator description
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.category._id
+      description: Category ID
+      type: number
+    - contextPath: FireEyeHX.Indicators.data.category.name
+      description: Category name
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.category.share_mode
+      description: Category share mode
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.category.uri_name
+      description: Category uri name
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.category.url
+      description: Category URL
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.uri_name
+      description: The indicator uri name
+      type: string
+    - contextPath: FireEyeHX.Indicators.data.stats.active_conditions
+      description: Indicator active conditions
+      type: number
+    - contextPath: FireEyeHX.Indicators.data.stats.alerted_agents
+      description: Total number of agents with HX alerts associated with this indicator.
+      type: number
+    - contextPath: FireEyeHX.Indicators.data.stats.source_alerts
+      description: Total number of source alerts associated with this indicator.
+      type: number
+    - contextPath: FireEyeHX.Indicators.data.update_actor._id
+      description: Update actor ID
+      type: number
+    - contextPath: FireEyeHX.Indicators.data.update_actor.username
+      description: Update actor name
+      type: string
+  description: Create new indicator
+- name: fireeye-hx-bulk-append-conditions-request
```

Comment:
improve the name, `bulk` and `request` shouldn't be in the command name, should be something like `append-conditions` or `add-conditions` - use FireEye UI terminology